zyte_common_items/__init__.py

```diff
@@ -12,3 +12,4 @@
     Metadata,
 )
 from .items import Product, ProductFromList, ProductList, ProductVariant
+from .page_objects import Page, ProductListPage, ProductPage
```

Comment:
What do you think about using "pages" convention for modules and packages with page objects?